website/src/react-native/showcase.js

```diff
@@ -622,6 +622,15 @@ var apps = [
     linkAppStore: 'https://itunes.apple.com/cn/app/hong-bei-bang-hai-liang-hong/id1007812319?mt=8',
     author: 'Hongbeibang'
   },
+  {
+    name: 'Reading',
+    icon: 'http://7xr0xq.com1.z0.glb.clouddn.com/about_logo.png',
+    linkPlayStore: 'http://www.wandoujia.com/apps/com.reading',
```

Comment:
he needs to change `linkPlayStore` key to be `link` key.

`linkPlayStore` is used to differentiate `ios` and `android` links.